src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -1157,6 +1162,19 @@ - (void)writeAttributeWithEndpointID:(NSNumber *)endpointID
                    timedWriteTimeout:(NSNumber * _Nullable)timeoutMs
                                queue:(dispatch_queue_t)queue
                           completion:(MTRDeviceResponseHandler)completion
+{
+    MTR_LOG("%@ write %@ 0x%llx 0x%llx: %@", self, endpointID, clusterID.unsignedLongLongValue, attributeID.unsignedLongLongValue, value);
```

Comment:
This would read better if it were write <cluster id> <attribute id> at <endpoint>

Reply:
This is matching the existing MTRDevice logging, which is sort of aiming for "log the path", which is an (endpoint, cluster, attribute) triple.